.gitlab/container_build/docker_linux.yml

```diff
@@ -4,17 +4,52 @@
   script:
     - TAG_SUFFIX=${TAG_SUFFIX:-}
     - BUILD_ARG=${BUILD_ARG:-}
+    - EXTRA_BUILD_CONTEXT=${ARTIFACTS_BUILD_CONTEXT:+"--build-context artifacts=$ARTIFACTS_BUILD_CONTEXT"}
```

Comment:
I didn't realize you could have multiple named contexts, and I have to agree that this makes the flow more readable